### DISCUSSION TITLE
Bruteforce

### DISCUSSION THREAD

**Discussion by Steeven-1002** [OpenGamePanel/OGP-Website#627] [General]
Bonjour,
Pourrais-je savoir si quelqu'un gère toujours le serveur d'opengamepanel.org ? Votre IP s'est faite bannir de mon serveur car vous avez tenté à plusieurs reprise de vous connecter en ssh avec l'utilisateur root. J'aimerais avoir des explications sur ce point.

**Comment by own3mall** [comment]
Provide proof.  Send logs.  Otherwise, this is just nonsense.  Our systems are not compromised as far as I know.

**Reply by Steeven-1002** [reply]
Merci pour votre réponse. Je viens de faire les captures d'écrans nécessaires.
![Screenshot 2024-02-03 100423](https://github.com/OpenGamePanel/OGP-Website/assets/121862054/711fbfa3-ef9b-43bd-b6d4-2a46f975af24)
![Screenshot 2024-02-03 100120](https://github.com/OpenGamePanel/OGP-Website/assets/121862054/ea118c0b-5c44-4b9a-9017-7328c3a026f1)
![Screenshot 2024-02-03 100308](https://github.com/OpenGamePanel/OGP-Website/assets/121862054/312cd389-0410-4f53-a77a-42d391cd9b6d)

**Reply by Steeven-1002** [reply]
Don't hesitate to ask me explaination of thoses screenshots if you want ;)

**Comment by Zorrototo** [comment]
Explain yourself yes.

This is not the OGP website IP. A quick DIG would give you the relevant information here. https://toolbox.googleapps.com/apps/dig/#ANY/opengamepanel.org

This has nothing to do with us here, anyone can install anything on their machine. What if it was a Discourse forum installed on the machine? Would you go to the Discourse project accusing them of BRUTE FORCE on your system?

**Reply by Steeven-1002** [reply]
Indeed, I apologize for not conducting thorough research and falsely "accusing" you. I want to clarify that my intention was simply to report potentially suspicious activities on your part, which does not appear to be the case now.

**Reply by Steeven-1002** [reply]
Besides the fact that it wasn't you, someone has apparently made a copy of your site, which is good to know. Instead of suggesting that I should be better because I'm supposed to be an engineer, we don't cover these aspects in our courses and I'm still a student, so yes, I didn't think to directly check your DNS, and I apologize for that. And yes, their IP is still online ;) 

As mentioned earlier, I didn't approach this with any ill intent; it was meant to be informative.

**Comment by own3mall** [comment]
It is not a "copy" of our website.  Open Game Panel (OGP) is a self hosted game server manager package.  It means anyone in the world anywhere can host an instance of the free software package we provide.  These instances have NOTHING to do or have any affiliation with us.  It could be running any website or software reachable on the IP or not.   Would you contact and complain to Cpanel for example if you saw their logo on that IP?  I hope not... it's just another popular package people run.  Please do better research next time, and please provide logs in the first place when making accusations.  Otherwise, it really wastes other people's time.

**Comment by Zorrototo** [comment]
Thanks for trying to report something, but what I meant is that you didn't search anything, went to an open source project repository on GitHub just because you saw the word OpenGamePanel on the page without searching anything. Nobody has made a copy of "our site" which is located here https://opengamepanel.org and which doesn't even look like the screenshot you shared.

Open Game Panel is an open source project anyone can install, like every single other open source project on GitHub (hence my comparison with another open source project like Discourse, or whatever anyone can install on his server).

//EDIT: posted at same time :D

**Comment by own3mall** [comment]
Also, just FYI, servers connected to the internet are attacked and probed pretty much all of the time by millions of IP addresses.  This is why it's recommended to use packages like fail2ban, iptables, and other software to help ban and manage theses IP addresses.  Also, if SSH is being brute forced attacked, change its default port to make it a little harder for hackers and exploiters.  Unfortunately, there are a lot of bad actors on the internet.  It's not uncommon to see a variety of attacks.  You should check IPs for abuse and report them here:  https://www.abuseipdb.com/

**Comment by Steeven-1002** [comment]
Thank you for your responses. I apologize once again for the inconvenience. I thought I was doing a good deed, but I didn't research the matter enough. Thanks for the website; it will be useful to me. I won't take up any more of your time. Best wishes to you.
